src/test/ui/suggestions/private-field.rs

```diff
@@ -0,0 +1,8 @@
+// compile-flags: --crate-type lib
+pub struct S {
+    pub val: String
```

Comment:
```suggestion
    pub val: MyString
```

What if the internal definition of `std::string::String` changes at some point and stops transitively containing a field named `cap`? Then the test will become useless. I suggest defining a custom type e.g. `MyString` to make it future-proof.

Reply:
Done.